core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -426,15 +426,15 @@ class ControllerBrokerRequestBatch(controller: KafkaController) extends  Logging
       case e : Throwable => {
         if (leaderAndIsrRequestMap.size > 0) {
           error("Haven't been able to send leader and isr requests, current state of " +
-              s"the map is $leaderAndIsrRequestMap")
+              s"the map is $leaderAndIsrRequestMap. Exception message: $e")
         }
         if (updateMetadataRequestMap.size > 0) {
           error("Haven't been able to send metadata update requests, current state of " +
-              s"the map is $updateMetadataRequestMap")
+              s"the map is $updateMetadataRequestMap. Exception message: $e")
         }
         if (stopReplicaRequestMap.size > 0) {
           error("Haven't been able to send stop replica requests, current state of " +
-              s"the map is $stopReplicaRequestMap")
+              s"the map is $stopReplicaRequestMap. Exception message: $e")
         }
         throw new IllegalStateException(e)
```

Comment:
@granthenke @ijuma One thing I observed in past logs I analyzed is that the exception is logged in the server log rather than the controller log. The "Haven't been able to..." message appears in the controller log. I find it more convenient to have at least the log message appearing along with the error message so that we can easily correlate the two.

I don't mind enforcing that the message is printed only once, which I think could happened when the method is called from one of the state machines (partition or replica), but first I'd like to know if you guys are happy with extending that log message in `ControllerChannelManager`.